targets/python/templates/Errors.py.ejs

```diff
@@ -10,12 +10,26 @@ class PlayFabErrorCode(Enum):
     <% } %><% var errorProps = errors[errorList[errorList.length-1]] %><%- errorProps.name %> = <%- errorProps.id %>
 
 class PlayFabError:
-    def __init__(self, errorMessage):
-        self.HttpCode = 0
-        self.HttpStatus = ""
-        self.Error = PlayFabErrorCode.Unknown # PlayFabError enum
-        self.ErrorMessage = errorMessage
-        self.ErrorDetails = {} # dictionary of string keys and list of strings for values
+
+    def __init__(self, *args):
+        if len(args) == 1:
+            self.fromJson(args[0])
+        else:
+            self.HttpCode = 0
+            self.HttpStatus = ""
+            self.Error = ""
+            self.ErrorCode = PlayFabErrorCode.Unknown
+            self.ErrorMessage = ""
+            self.ErrorDetails = {}# dictionary of string keys and list of strings for values
```

Comment:
Should default to None
Many errors from PlayFab don't necessarily return ErrorDetails.